src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styleDeclarations = sourceString.split(';');
+
+  styleDeclarations.forEach((style) => {
+    const parts = style.split(':');
+
+    if (parts.length === 2) {
+      const property = parts[0].trim();
+      const value = parts[1].trim();
+
+      styleObject[property] = value;
+    }
+  });
```

Comment:
The 'forEach' loop here is appropriate, but you can make the code more functional and concise by using 'reduce' to build up the 'styleObject'. This would avoid the need to declare an empty object at the start and mutate it inside the loop.

Reply:
Great suggestion from AI